deploy/deploy.go

```diff
@@ -461,6 +466,16 @@ func walkLocal(fs afero.Fs, matchers []*matcher) (map[string]*localFile, error)
 			path = norm.NFC.String(path)
 		}
 
+		// Check include/exclude matchers.
+		if include != nil && !include.Match(path) {
```

Comment:
I notice you're doing the ToSlash lower down. The Glob matches on Unix style paths, so I don't think this would work in the non-simple cases on Windows.

Reply:
Good catch! Fixed to match against `slashpath`.